docs/FRSKY_R9.md

```diff
@@ -21,31 +21,45 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
   </tbody>
 </table>
 
-Comment: mLRS also supports the R9MM receiver. However, flashing the R9MM with ST-Link is really tedious and requires top soldering skills or employing other tricks, as one needs to connect to four tiny solder pads. You really should consider using the R9MX receiver instead. Note: The R9MM has a different wiring diagram than the R9MX.
-
 ## R9M Tx Module ##
 
-The R9M module is unfortunately limited with respect to serial ports. It provides acess to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
+The R9M module is somewhat limited with respect to serial ports. It provides access to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
+
+1. One needs to use a "Frsky inverter" dongle. You either can buy one or build it yourself.  Alternatively, if you will use the Lua script for configuration, and wish to use the serial port to connect via MAVLink wirelessly to a Ground Control Station, you can use the one of the supported esp32 boards with the mlrs-wifi-bridge sketch.
```

Comment:
since it's first time mentioned, maybe "... mLRS Lua configuration script for ..."

EPS32 (we try to use the "official" acronyms, if there are any)

The "Alterrnatively" and givern condition reads wrong to me. Like "Eat a bannana or go swimming" and not "Eat a bannana or an apple". The Lua would refer to using the serial port as COM/CLI. I guess I would discuss the use options elsewhere, and here maybe only write "You also can use one of the supported ESP32 boards with the mlrs-wifi-bridge sketch, as it can do the signal inversion"